scripts/spin.ps1

```diff
@@ -124,6 +124,8 @@ if ($CodeCoverage) {
 $SpinQuic = $null
 if ($IsWindows) {
     $SpinQuic = Join-Path $RootArtifactDir "spinquic.exe"
+    # Set ASAN option to prevent deadlock bug in ASAN RtlReAllocHeap.
+    $env:ASAN_OPTIONS='windows_hook_legacy_allocators=1'
```

Comment:
This won't actually set the env for the process that gets started. In essence this does nothing.

Reply:
What you instead need to do is in run-executable in the Start-Executable function after you create $pinfo is call `$pinfo.Environment.Add(key, value)`